conwayste/src/client.rs

```diff
@@ -386,7 +386,7 @@ impl MainState {
             GameError::ConfigError(msg)
         })?;
 
-        let ui_layout = UILayout::new(ctx, &config, font.clone());
+        let ui_layout = UILayout::new(ctx, &config, font.clone()).unwrap();
```

Comment:
Please either fix the `.unwrap()` so this can't panic or explain why it can't happen in a comment. :)